.gitlab-ci.yml

```diff
@@ -33,6 +33,7 @@ variables:
     - rustup show
     - bash --version
     # global RUSTFLAGS overrides the linker args so this way is better to pass the flags
```

Comment:
Comment should be moved 1 line down.

Reply:
actually `mkdir` and `touch` fix the previous problem we had with writing to `CARGO_HOME` for the first time, but ok.